### DISCUSSION TITLE
Upgrading 13.0 to 13.5

### DISCUSSION THREAD

**Discussion by BGillham** [2sic/2sxc#2709] [Q&A]
Can we upgrade v13.0 to 13.5 without any "issues" or are there some "adjustments" to make?

**Comment by iJungleboy** [comment]
Almost. 
ATM any features you had activated, must be re-activated manually.
13.06 will fix this. 
Otherwise there are no known adjustments.
In very rare cases (like if you had very deep-code like custom data sources) you will need to check the breaking changes, but for any normal website there should be nothing to adjust at all.
